<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScene</class>
 <widget class="QMainWindow" name="MainScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainScene</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Start</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="actionStart_BGM"/>
    <addaction name="actionStop_BGM"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuStart"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStop_BGM">
   <property name="text">
    <string>Stop BGM</string>
   </property>
  </action>
  <action name="actionStart_BGM">
   <property name="text">
    <string>Start BGM</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
